### PR TITLE
[SeaTunnel#1100]Add SkyWalking LICENSE

### LINKED ISSUE #1100: [License Issue] Please update your source codes' license
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

I accidentally noticed this PR, https://github.com/apache/incubator-seatunnel/pull/912/files#diff-a0275ef336fd0cf14c3d4c026ce3e918301dc0cf7b2663fbf484d9cff5288bd2, which carried the dependency license check script into your source codes. And the same style of `known-dependencies.txt` file, which was created by SkyWalking.

But you should know, this script is coming(copied) from [Apache SkyWalking](https://github.com/apache/skywalking/blob/master/tools/dependencies/check-LICENSE.sh), which was written by one PMC member. And I noticed this script was not submitted by himself(he still owns this file). This file's owner has reviewed this PR, but I think he missed this. Unless he is the one to submit, the License should be taken care of.

So, you should update the source codes' LICENSE to indicate you have copied it from another project. 

FYI @CalvinKirs @zhongjiajie

### SeaTunnel Version

The dev branch

### SeaTunnel Config

```conf
No need.
```

### Running Command

```shell
No need.
```

### Error Exception

```log
No need.
```

### Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by CalvinKirs**
this closes #1100

### CODE REVIEW

**Comment by CalvinKirs** [comment]
@wu-sheng hi, please help to see, thanks

File: seatunnel-dist/release-docs/LICENSE [hunk 1]
```diff
@@ -693,6 +693,7 @@ The text of each license is the standard Apache 2.0 license.
      (The Apache Software License, Version 2.0) Commons Logging (commons-logging:commons-logging:1.1.3 - http://commons.apache.org/proper/commons-logging/)
      (Apache 2) Objenesis (org.objenesis:objenesis:2.5.1 - http://objenesis.org)
      (The Apache Software License, Version 2.0) Apache Commons Math (org.apache.commons:commons-math3:3.5 - http://commons.apache.org/proper/commons-math/)
+     (The Apache Software License, Version 2.0) Apache SkyWalking : 8.9.0 (https://github.com/apache/skywalking/)
```
Comment (wu-sheng) [inline seatunnel-dist/release-docs/LICENSE]: Do you include this script into the binary tar? If not, this license doesn't need to update.

Comment (CalvinKirs) [reply seatunnel-dist/release-docs/LICENSE]: yup, we don't need to keep this in binary tar, removed

**Review by wu-sheng — APPROVED** [review]

**Comment by CalvinKirs** [comment]
Thanks:)

**Review by zhongjiajie — APPROVED** [review]

**Comment by zhongjiajie** [comment]
Thanks

**Pull request merged**